HW02/aleksandr_serikov/array_refinements.rb

```diff
@@ -0,0 +1,35 @@
+require 'pry'
+
+module ArrayRefinement
+
+  refine Array do
+  
+    def each
+      for i in 0...size
+        yield self[i] 
+      end
+      self
+    end
+    
+    def map
+      new_array = []
+      for i in 0...size
+        new_array << (self[i] && yield(self[i]))
```

Comment:
Why do you need it? `1 && 2` => `2`
```suggestion
        new_array <<  yield(self[i])
```

Reply:
Done